app/controllers/restaurants_controller.rb

```diff
@@ -65,13 +65,12 @@ def destroy
   end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
     def set_restaurant
       @restaurant = Restaurant.find(params[:id])
     end
 
-    # Never trust parameters from the scary internet, only allow the white list through.
+    # require params methods
     def restaurant_params
-      params.require(:restaurant).permit(:name, :rating, :tenbis, :address, :price, :maxDevTime,:cuisine_id)
+      params.require(:restaurant).permit(:name, :rating, :tenbis, :address, :kosher, :maxDevTime,:cuisine_id)
```

Comment:
in ruby we the syntax should be with under_score and not camelCase :)

Reply:
`sssssnake_case`